src/RoomsSDKAdapter.test.js

```diff
@@ -16,6 +16,8 @@ describe('Rooms SDK Adapter', () => {
       type: 'group',
       title: 'mock room',
     };
+
+    mockSDK.rooms.get = jest.fn(() => Promise.resolve(mockSDKRoom));
```

Comment:
Is there a reason why this is not defined within the mock and done here?
`mockSDKRoom` could be exported if necessary (i think only one test would need it if removed from here)